cmd/cmd-run.go

```diff
@@ -60,9 +60,15 @@ Available values:
   replace: Replace the existing content of the branch by force pushing any new changes, then reuse any existing pull request, or create a new one if none exist.
 `)
 	cmd.Flags().BoolP("draft", "", false, "Create pull request(s) as draft.")
+	cmd.Flags().BoolP("pr-auto-merge", "", false, "Enable auto-merge for created pull requests. PRs will be automatically merged when all required checks pass (GitHub) or when pipeline succeeds (GitLab). Use --merge-type to specify the merge strategy for GitHub.")
+	cmd.Flags().StringSliceP("merge-type", "", []string{"merge", "squash", "rebase"},
+		"The type of merge that should be done (GitHub/Gitea). Multiple types can be used as backup strategies if the first one is not allowed. The first type is used for auto-merge.")
```

Comment:
This is re-defined together with the `RegisterFlagCompletionFunc` (it is also defined for the merge command). Please create a function to share this definition. Maybe with a parameter to indicate if "Multiple types can be used as backup strategies if the first one is not allowed." should be added or not, so that "The first type is used for auto-merge." can be removed.